openassessment/xblock/static/js/spec/studio/oa_edit_fields.js

```diff
@@ -159,3 +159,147 @@ describe("OpenAssessment.ToggleControl", function() {
     });
 
 });
+
+
+describe("OpenAssessment.SelectControl", function() {
+
+    var StubNotifier = function() {
```

Comment:
Can we share this method with `OpenAssessment.ToggleControl`?  They look similar to me.